firmware/Reload Pro.cydsn/ui.c

```diff
@@ -108,13 +112,24 @@ const menudata choose_readout_menu = {
 	}
 };
 
+const menudata tools_menu_data = {

```

Comment:
What would your preference for the function for the tools menu be?
I chose that name to avoid conflicts, as i felt it was more natural to call the menu function `tools_menu` ?